CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 0.15.0
+## Added
+* Support AVM 7 updates:
+  * New opcodes:
+    * base64_decode` ([#418](https://github.com/algorand/pyteal/pull/418))
```

Comment:
```suggestion
    * `base64_decode` ([#418](https://github.com/algorand/pyteal/pull/418))
```
Missing opening tick?

Reply:
@jasonpaulos Thanks for catching the formatting issue.